cli/package.json

```diff
@@ -92,6 +92,7 @@
     "dependency-check": "3.3.0",
     "dtslint": "0.7.1",
     "execa-wrap": "1.4.0",
+    "hasha": "5.0.0",
```

Comment:
is this intentional

Reply:
yup, code was lifted, but dev dependency to validate inside a test